index.bs

```diff
@@ -676,6 +681,27 @@ Issue: Parse |value|["`filter_data`"].
 
 Issue: Parse |value|["`aggregation_keys`"].
 
+<h3 dfn id="should-block-processing-for-destination-limit">Should processing be blocked by destination limit</h3>
+
+Given an [=attribution source=] |source|:
+1. Let |pendingSources| be all entries in the [=attribution source cache=] where all of the following are true:
+     * entry's [=attribution source/source site=] and |source|'s [=attribution source/source site=]
+        are [=same site=].
+     * entry's [=attribution source/reporting endpoint=] and |source|'s
+        [=attribution source/reporting endpoint=] are [=same origin=].
```

Comment:
Elsewhere in the spec, we compare this field by saying "equal" rather than "same origin". If those uses are wrong, could we update them first? If not, let's be consistent and say "equal".